README.md

```diff
@@ -16,7 +18,169 @@ We use Docker for development purposes. To start working, your computer needs to
 | `make test`  | runs tests     |
 | `make lint`  | runs linters   |
 
-Type: `less Makefile` to see other commands
+Type `make help` to see other commands.
 
 ### Default User password
-'pass123'
+
+`pass123`
+
+### How to debug the app with remote database connection
+
+---
+> Please note that this step is optional.
+> You most likely do not need to follow it.
+---
+
+Here's some information on how to set up the environment
+so that you will be connected to dev DB or produciton
+one using your local docker-compose setup.
+
+</br>
+
+#### 1. Copy `.envrc.example` to `.envrc`
+
+This is needed to avoid making changes to the
+`.envrc.example` file which could be commited by
+accident, because it's not ignored.  
+On the other hand, `.envrc` is ignored,
+so it's safe to modify it without ever worrying about
+commiting it by accident.
+
+
+```bash
+# run this in terminal, in the project directory
+cp .envrc.example .envrc
+```
+
+</br>
+
+#### 2. Edit the `.envrc` file using your favourite text editor
+
+Replace the secret values (passwords and secrets).
```

Comment:
What is a secret in .envrc file? There is no field secret isn't it?

Reply:
Oh, you got me there. I used `secrets` to refer to basically every key in `.envrc.example`. I consider both passwords as well as usernames to be 'secret' - as in - "it shouldn't be known, even if the value is not surprising".